### ISSUE TITLE
Docker Quickstart doesn't work

### ISSUE THREAD

**Issue by bearrito** [latenighttales/alcali#500]
```
git clone https://github.com/latenighttales/alcali.git
cd alcali
docker-compose up --scale minion=2
```

```
 File "/opt/saltstack/salt/lib/python3.10/site-packages/salt/cli/run.py", line 34, in run
    ret = runner.run()
  File "/opt/saltstack/salt/lib/python3.10/site-packages/salt/runner.py", line 300, in run
    ret = self._proc_function(
  File "/opt/saltstack/salt/lib/python3.10/site-packages/salt/client/mixins.py", line 540, in _proc_function
    return instance.low(fun, low)
  File "/opt/saltstack/salt/lib/python3.10/site-packages/salt/client/mixins.py", line 418, in low
    salt.utils.job.store_job(
  File "/opt/saltstack/salt/lib/python3.10/site-packages/salt/utils/job.py", line 76, in store_job
    raise KeyError(emsg)
KeyError: "Returner 'alcali' does not support function prep_jid"
ERROR: Service 'master' failed to build: The command '/bin/sh -c salt-run saltutil.sync_all' returned a non-zero code: 1
```
```
bstrausser@LOCLAP680:~/Git/alcali$ git branch
* develop
```

```
bstrausser@LOCLAP680:~/Git/alcali$ git log
commit 4a652bc421f73f54e9dd3fcb3f46d3999c977111 (HEAD -> develop, origin/develop, origin/HEAD)
Author: Matt Melquiond <matt.LLVW@gmail.com>
Date:   Sat Jun 25 22:04:42 2022 +0200

    add a clear results button (#455)

commit 0aa8d0fbbecfb11b4346d47aa4f9f3cabd7f0fe0

```

**Comment by kragebein** [comment]
Getting the same using the salt formula

**Comment by agriev** [comment]
same here

**Comment by agriev** [comment]
Just for the information, alcali has some problems with a folder permissions in docker.

quickfix is in entrypoint-master.sh:

mkdir /var/cache/salt/master
chown -R salt:salt /var/cache/salt

salt-run saltutil.sync_all # remove it from the Dockerfile-master

Also in Dockerfile-master:

RUN salt-call --local tls.create_self_signed_cert cacert_path='/etc/pki'
RUN chmod -R a+rwx /etc/pki/. # add this line to fix certificates permission

**Comment by mattLLVW** [comment]
fixed by #507
